daemon/cluster/listen_addr.go

```diff
@@ -123,6 +123,25 @@ func validateDefaultAddrPool(defaultAddrPool []string, size uint32) error {
 	return nil
 }
 
+// getDataPathPort validates vxlan udp port (data path port) number.
+// if no port is set, the default (4789) is returned
+// valid port numbers are between 1024 and 49151
+func getDataPathPort(portNum uint32) (uint32, error) {
+	// if the value comes as 0 by any reason we set it to default value 4789
+	if portNum == 0 {
+		portNum = 4789
```

Comment:
~Did something go wrong with a rebase? I thought you added a `const` for the default port 🤔~

edit: ah; never mind, that was in libnetwork https://github.com/docker/libnetwork/blob/87b1efc194575f94618d1185e808c4794f3d1c3d/drivers/overlay/overlayutils/utils.go#L14-L27